### ISSUE TITLE
What do I download?

### ISSUE THREAD

**Issue by notNSANE** [Taiizor/Sucrose#108]
Can you please just upload a zip for windows 11 x64?

there are like 30 .net versions and I have no idea what to pick.

**Comment by Taiizor** [comment]
Yes, this looks quite confusing. Over time, I removed two different versions, but I will continue to publish them for a while longer to avoid causing issues for current users. If you are confused, you can download the application either from [Microsoft Store](https://apps.microsoft.com/detail/XP8JGPBHTJGLCQ?launch=true&mode=full) or for [.NET Framework 4.8 (x64)](https://github.com/Taiizor/Sucrose/releases/download/v24.12.2.0/Sucrose_Bundle_.NET_Framework_4.8_x64_24.12.2.0.exe).

I am releasing the application for three different architectures and six different frameworks (.NET). Different architectures are, of course, necessary. As for the frameworks, different frameworks require different dependencies. Therefore, if you choose the latest frameworks, you will also need to install them on your computer to use the application. For this reason, I preferred the [.NET Framework 4.8](https://dotnet.microsoft.com/en-us/download/dotnet-framework/net48) version on the [Microsoft Store](https://apps.microsoft.com/detail/XP8JGPBHTJGLCQ?launch=true&mode=full), as it does not require additional dependencies (since it is already included in Windows). If you choose the more recent framework versions, you can achieve better performance.

**Comment by Taiizor** [comment]
I have removed .NET 6 and .NET 7 versions. Hopefully, this will at least cause less confusion.

**Comment by Taiizor** [comment]
I have also removed .NET Framework 4.8.1 and .NET 8 versions. Hopefully, this will at least cause less confusion.
